_posts/2021-06-08-moveit-vs-moveit2.md

```diff
@@ -46,21 +56,31 @@ With the release of ROS 2 Galactic, we would like to share the new features only
     <tr>
       <td>Setup Assistant</td>
       <td class="done">✓</td>
-      <td class="dev">In Development</td>
+      <td class="dev">in development</td>
+    </tr>
+    <tr>
+      <td>MoveIt Task Constructor</td>
+      <td class="done">✓</td>
+      <td class="done">active fork</td>
     </tr>
     <tr>
       <td>Game Controller Integration for Servo</td>
       <td class="done">✓</td>
       <td class="done">✓</td>
     </tr>
+    <tr>
+      <td>Industrial Trajectory Generator</td>
+      <td class="done">✓</td>
+      <td class="dev">planned</td>
+    </tr>
     <tr>
       <td><a href="https://github.com/ros-planning/moveit2/tree/main/moveit_demo_nodes/run_ompl_constrained_planning" target="_blank">Probabilistically complete Cartesian Planning</a></td>
-      <td class="not">✕</td>
+      <td class="dev"><a href="https://github.com/ros-planning/moveit/pull/2273" target="_blank">Stale Patch</a></td>
       <td class="done">✓</td>
     </tr>
     <tr>
-      <td>ROS 2 Component Nodes</td>
-      <td class="not">✕</td>
+      <td>Composable Nodes</td>
+      <td class="dev">nodelet subsystem</td>
       <td class="done">✓</td>
```

Comment:
The `mesh_filter` is implemented as a nodelet. Granted, it's not the most prominent part of the code base.
I guess if anyone were interested in it, providing a `moveit_servo` nodelet is a matter of a few hours (which makes it a "one foot in the former" item from Gijs previous explanations :smile: ).

Reply:
I didn't know about the mesh_filter.  I had a PR to make servo a nodlet once but closed it because at the same time I did a large refactor and a bunch of stuff changed and I never rebased the nodelet PR.